css/src/theme/system-admin.css

```diff
@@ -0,0 +1,11 @@
+/**
+ * @file
+ * Styles for administration pages.
+ */
+
+/**
+ * Appearance page.
+ */
+.system-themes-list-uninstalled {
```

Comment:
Can we add a todo that this will be removed after https://github.com/drupalux/seven/issues/58 has been solved?

Reply:
@todo We need to remove css/src/theme/system-admin.css this file after #58 has been solved.